LFPy/test/test_eegmegcalc.py

```diff
@@ -230,7 +299,25 @@ def test_calc_potential(self):
 
         np.testing.assert_allclose(pots_4s, pots_inf, rtol=1e-6)
 
+    def test_calc_potential01(self):
+        '''test comparison between analytical 4S-model and FEM simulation'''
+        # load data
+        fem_sim = np.load(os.path.join(LFPy.__path__[0], 'test', './fem_mix_dip.npz'))
```

Comment:
`os.path.join(LFPy.__path__[0], 'test', 'fem_mix_dip.npz')` should also work on windows